builder/bundles.go

```diff
@@ -364,13 +364,39 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 }
 
 func installFilesystem(packagerCmd []string, chrootDir string) error {
-	installArgs := merge(packagerCmd,
-		"--installroot="+chrootDir,
-		"install",
-		"filesystem",
-	)
-	_, err := helpers.RunCommandOutputEnv(installArgs[0], installArgs[1:], []string{"LC_ALL=en_US.UTF-8"})
-	return err
+	_, err := downloadRpms(packagerCmd, []string{"filesystem"}, chrootDir, 1)
+	if err != nil {
+		return err
+	}
+	path := chrootDir + "/var/cache/yum/clear/"
```

Comment:
We should avoid hardcode paths, specially ones that repeat in other places. Put this into a variable or constant.